examples/create-and-alter-table/migrations/1-create-users-table.yml

```diff
@@ -0,0 +1,22 @@
+version: 0.0.1
+
+up:
+  table: users
+  operation: create
+  runner: mysql
+  query: >
+  CREATE TABLE users (
+    id INT(6) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    firstname VARCHAR(30) NOT NULL,
+    lastname VARCHAR(30) NOT NULL,
+    email VARCHAR(50),
+    reg_date TIMESTAMP
+  )
+
+down:
+  table: users
+  operation: delete
```

Comment:
Might be for later, but what is the list of possible operations? The query is `DROP` but the operation is `delete`?

Reply:
Yeah, sent it as an example. 
attached issue for it